tests/fixtures/user-batchupdate-event-payload.json

```diff
@@ -0,0 +1,314 @@
+{
+    "notification_id": "cc156fd4-5150-4f1f-908f-63287beafe5d",
+    "configuration": {
+        "id": "5bbb8592ec9ecaa580002cff",
+        "organization": "ad793dc7.hullapp.io",
+        "secret": "7640d59e15c5959e599ef437e3ba248d"
+    },
+    "connector": {
+        "tags": [],
+        "source_url": "https://d9bb86b5.ngrok.io/",
+        "private_settings": {
+          "synchronized_segments": [
+            "5ade25df4d257947aa001cd5"
+        ],
+          "synchronized_properties": [
+            "last_seen_at"
+        ]},
+        "index": "https://d9bb86b5.ngrok.io/ship.js",
+        "name": "Tims Local Ngrok",
+        "extra": {},
+        "settings": {
+            "public_id_field": "external_id",
+            "handle_pages": true,
+            "handle_screens": true,
+            "handle_accounts": true,
+            "ignore_segment_userId": false,
+            "public_account_id_field": "external_id",
+            "handle_groups": false,
+            "write_key": "fakekey"
+        },
+        "type": "ship",
+        "manifest": {
```

Comment:
maybe let's not embed acutal manifest data in the example payloads as they are eventually going to get out of sync with actual manifest. If we need the manifest for testing purposes, then let's load it from the actual manifest.json file in the repo and inject it here at runtime, otherwise I would just put an empty manifest definition in the test payload.

Reply:
that's fine, makes sense, though it's nice having clear input